src/index.html

```diff
@@ -25,6 +25,9 @@
       </div>
       <div id="modal"></div>
   </div>
+  <div id="text-area"></div>
+  <div id="modal"></div>
+  <div id="checkbox"></div>
```

Comment:
Since I changed the layout a bit, I think this needs to be moved up inside the #`bottom` div, otherwise it's not showing up.

If you put it in line 24 between `#bottom` and `#compose-area` it should work